docs/content/features/exploiters/powershell.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "PowerShell Exploiter"
+draft: false
+description: "Exploits PowerShell Remoting"
+tags: ["exploiter", "powershell", "brute force"]
+pre: "<i class='fa fa-terminal'></i> "
+---
+
+## PowerShell Remoting
+
+[PowerShell Remoting](
+https://docs.microsoft.com/en-us/powershell/scripting/learn/remoting/winrmsecurity)
+is a PowerShell feature that enables connecting to Windows machines remotely in
+order to run arbitrary commands. It is commonly used by administrators to
+manage multiple systems in a network.
+
+## Exploitation
+
+Machines with PowerShell Remoting enabled may be accessible to attackers if
+they come across the correct credentials.
```

Comment:
In general, I'm not sure I like the language, "if they come across the correct credentials." We should maybe come up with something that's a bit more explanatory.

Reply:
https://github.com/guardicore/monkey/pull/4269#discussion_r1705247834